ocs_ci/ocs/ui/base_ui.py

```diff
@@ -0,0 +1,152 @@
+import os
+import shutil
+
+from zipfile import ZipFile
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.common.by import By
+
+
+from ocs_ci.utility.utils import run_cmd, get_kubeadmin_password
+from ocs_ci.ocs.ui.views import login
+from ocs_ci import framework
+
+
+logger = logging.getLogger(__name__)
+
+
+class BaseUI:
+    """
+    Base Class for UI Tests
+
+    """
+
+    def __init__(self, driver):
+        self.driver = driver
+
+    def do_click(self, by_locator, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((type, by_locator)))
+        element.click()
+
+    def do_click_visibility(self, by_locator, type=By.PARTIAL_LINK_TEXT, timeout=30):
+        wait = WebDriverWait(self.driver, timeout=timeout)
+        element = wait.until(ec.visibility_of_element_located((type, by_locator)))
+        element.click()
+
+    def do_send_keys(self, by_locator, text, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((type, by_locator)))
+        element.send_keys(text)
+
+    def get_element_text(self, by_locator, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.visibility_of_element_located((type, by_locator)))
+        return element.text
+
+    def is_enabled(self, by_locator, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.visibility_of_element_located((type, by_locator)))
+        return bool(element)
+
+    def get_title(self, title, type=By.XPATH, timeout=30):
+        wait = WebDriverWait(self.driver, timeout)
+        wait.until(ec.title_is(type, title))
+        return self.driver.title
+
+
+def install_chromedriver():
+    """
+    Install chromedriver
+
+    """
+    bin_dir = framework.config.RUN.get("bin_dir")
+    files_bin = os.listdir(bin_dir)
+    if "chromedriver" in files_bin:
+        return True
+    tmp_dir_path = tempfile.mkdtemp()
+    url = "https://chromedriver.storage.googleapis.com/87.0.4280.20/chromedriver_linux64.zip"
+    logger.info(f"download chromedriver {url}")
+    wget.download(url, tmp_dir_path)
+    files = os.listdir(tmp_dir_path)
+    chromedriver_file = os.path.join(tmp_dir_path, files[0])
+    zip = ZipFile(chromedriver_file)
+    logger.info(f"Unzip {chromedriver_file}")
+    zip.extractall(path=tmp_dir_path)
+    os.chmod(os.path.join(tmp_dir_path, "chromedriver"), 777)
+    shutil.move(os.path.join(tmp_dir_path, "chromedriver"), bin_dir)
+
+
+def login_ui(browser):
+    """
+    Login to OpenShift Console
+
+    Args:
+        browser(str): type of browser (chrome, firefox..)
+
+    return:
+        driver(Selenium WebDriver)
+
+    """
+    install_chromedriver()
```

Comment:
If browser is chrome, that's correct, but if browser is firefox, chromedriver shouldn't be installed.